contracts/bridge/Bridge.sol

```diff
@@ -299,17 +359,35 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
     function chargeWithoutEvent() external payable {}
 
     // setKLAYFee set the fee of KLAY tranfser
-    function setKLAYFee(uint256 _fee) external onlyOwner {
+    function setKLAYFee(uint256 _fee, uint64 _requestNonce)
+        external
+        onlyOperators {
+        onlySequentialNonce(VoteType.Configuration, _requestNonce);
+        bytes32 voteKey = keccak256(abi.encodePacked(this.setKLAYFee.selector, _fee, _requestNonce));
+        if (!voteConfiguration(voteKey, msg.sender)) {
+            return;
+        }
         _setKLAYFee(_fee);
     }
 
     // setERC20Fee set the fee of the token transfer
-    function setERC20Fee(address _token, uint256 _fee) external onlyOwner {
+    function setERC20Fee(address _token, uint256 _fee, uint64 _requestNonce)
+        external
+        onlyOperators
+    {
+        onlySequentialNonce(VoteType.Configuration, _requestNonce);
+        bytes32 voteKey = keccak256(abi.encodePacked(this.setERC20Fee.selector, _token, _fee, _requestNonce));
+        if (!voteConfiguration(voteKey, msg.sender)) {
```

Comment:
We can deal with `onlySequentialNonce(VoteType.Configuration, _requestNonce);` in voteConfiguration if we pass requestNonce to it.